libraries/grpc-sdk/src/interfaces/Model.ts

```diff
@@ -49,4 +71,52 @@ export interface ConduitSchemaOptions {
       canDelete: boolean;
     };
   };
+  indexes?: ModelOptionsIndexes[];
+}
+
+export interface SchemaFieldIndex {
+  type?: MongoIndexType | PostgresIndexType;
+  options?: MongoIndexOptions | PostgresIndexOptions;
+  [field: string]: any;
+}
+
+export interface ModelOptionsIndexes {
+  fields: string[];
+  types?: MongoIndexType[] | PostgresIndexType;
```

Comment:
No, in case of a compound index, in mongo you have to define an index type value for every field (e.g. {email: 1, password: 1}) but in postgres you define one type value that represents the entire index (e.g. { fields: ['email', 'password'], using: 'BTREE'}).